src/playercards/cards/KeyofSolomon.ttslua

```diff
@@ -0,0 +1,135 @@
+require("playercards/CardsWithHelper")
+local blessCurseManagerApi     = require("chaosbag/BlessCurseManagerApi")
+local playermatApi             = require("playermat/PlayermatApi")
+
+-- intentionally global
+hasXML                         = true
+isHelperEnabled                = false
+local updated, loopId
+
+function updateSave()
+  self.script_state = JSON.encode({
+    isHelperEnabled = isHelperEnabled,
+    loopId = loopId
+  })
+end
+
+function onLoad(savedData)
+  if savedData and savedData ~= "" then
+    local loadedData = JSON.decode(savedData)
+    isHelperEnabled = loadedData.isHelperEnabled
+    loopId = loadedData.loopId
+  end
+  if isHelperEnabled then updateDisplay() end
+end
+
+function shutOff()
+  if loopId then
+    Wait.stop(loopId)
+    loopId = nil
+  end
+end
+
+function initialize()
+  maybeUpdateButtonState()
+  loopId = Wait.time(maybeUpdateButtonState, 1, -1)
+end
+
+function resolveToken(player, _, tokenType)
+  if not updated then return end
+
+  local matColor
+  if player.color == "Black" then
+    matColor = playermatApi.getMatColorByPosition(self.getPosition())
+  else
+    matColor = playermatApi.getMatColor(player.color)
+  end
+
+  blessCurseManagerApi.removeToken(tokenType, player.color)
+
+  if tokenType == "Curse" then
+    local actionResources = 2
+    playermatApi.updateCounter(matColor, "ResourceCounter", nil, actionResources)
+    printToColor("Added " .. actionResources .. " resources to " .. matColor .. "'s resource pool.", player.color)
```

Comment:
I think this should use `GlobalApi.getColoredName(playerColor)`.
And also probably print it for both the triggering player and the effected player (if different).